internal/controller/nonadminbackup_controller.go

```diff
@@ -51,6 +51,7 @@ type NonAdminBackupReconciler struct {
 const (
 	phaseUpdateRequeue     = "NonAdminBackup - Requeue after Phase Update"
 	conditionUpdateRequeue = "NonAdminBackup - Requeue after Condition Update"
+	statusUpdateError      = "Failed to update NonAdminBackup Status"
```

Comment:
Lets use this `statusUpdateError` string everywhere. Lets drop usage of `phaseUpdateError` and `conditionUpdateError` now that we are combining calls. `statusUpdateError` make more sense IMO.

Reply:
will be applied in next commit